pkg/acceptance/bettertestspoc/assert/resourceassert/account_resource_gen.go

```diff
@@ -195,3 +195,152 @@ func (a *AccountResourceAssert) HasNoRegionGroup() *AccountResourceAssert {
 	a.AddAssertion(assert.ValueNotSet("region_group"))
 	return a
 }
+
+////////////////////////////
+// Attribute empty checks //
+////////////////////////////
+
+func (a *AccountResourceAssert) HasAdminNameEmpty() *AccountResourceAssert {
```

Comment:
let's filter out the needed messages by the "required" attribute in schema - e.g. this can never be empty, so this assertion should not be generated